nameko/constants.py

```diff
@@ -1,5 +1,5 @@
 AMQP_URI_CONFIG_KEY = 'AMQP_URI'
-WEB_SERVER_CONFIG_KEY = 'WEB_SERVER_ADDRESS'
+WEB_SERVER_CONFIG_KEY = 'WEB_SERVER_URI'
```

Comment:
while this may be a better key, i'm not sure it's better _enough_ to justify breaking backwards compatibility